Dockerfile

```diff
@@ -9,16 +9,17 @@ ARG REACT_APP_SENTRY_KEY=${REACT_APP_SENTRY_KEY:-""}
 ARG REACT_APP_SENTRY_ORGANIZATION=${REACT_APP_SENTRY_ORGANIZATION:-""}
 ARG REACT_APP_SENTRY_PROJECT=${REACT_APP_SENTRY_PROJECT:-""}
 
-RUN npm i -g yarn
+RUN npm i -g ionic
```

Comment:
same

Reply:
actually we don't need changes to Dockerfile, because Android app is an executable file, not an image
reset changes to Dockerfile